tests/www/LoginIntegrationTest.php

```diff
@@ -2,8 +2,10 @@
 
 namespace Simplesamlphp\Casserver;
 
-use GuzzleHttp\Client;
-use GuzzleHttp\Cookie\CookieJar;
+
+use \SimpleSAML\Test\BuiltInServer;
+
+include(dirname(__FILE__) . '/../../vendor/simplesamlphp/simplesamlphp/tests/BuiltInServer.php');
```

Comment:
Why do you manually include it here? Shouldn't be necessary

Reply:
It fails I do not include it.

Reply:
Ah, makes sense.. It should be included [here](https://github.com/simplesamlphp/simplesamlphp/blob/d2960c0c295e570476b715fe5a2918cb66feb880/tests/bootstrap.php#L7) to make it work without the include..